### PR TITLE
last fixes for release of v0.8

### PR DESCRIPTION
**Pull request by kosack**
- remove pip installs from environment.yml, which are now conda packages (for Linux and MacOS)

### CODE REVIEW

**Comment by kosack** [comment]
any other last-minute cleanups?

**Comment by MaxNoe** [comment]
see #1349

File: environment.yml [hunk 1]
```diff
@@ -42,6 +42,5 @@ dependencies:
   - xz
   - zlib
   - zstandard
-  - pip:
-    - eventio~=1.0
-    - https://github.com/cta-observatory/ctapipe-extra/archive/v0.3.0.tar.gz
+  - eventio>=1.0
```
Comment (MaxNoe) [inline environment.yml:45]: we require at least  1.1.1 for the calibration event type. I thought I updated this also here (in setup.py it's correct)

Comment (MaxNoe) [reply environment.yml:45]: change to `>=1.1.1,<2`

Comment (kosack) [reply environment.yml]: done.  For the conda package, all specs are taken from setup.py, so if a user installs it via conda, it will be all ok.

**Pull request merged**